backstop/scenarios/modal.js

```diff
@@ -6,13 +6,22 @@ module.exports = [
     // when "Open modal" sometimes is and sometimes is not hovered.
     actions: [click('#open-modal'), hover('body')],
     selectors: ['document'],
-    hideSelectors: ['body > *:not(.iui-modal)'],
+    hideSelectors: ['body > *:not(.iui-modal-backdrop)'],
   }),
+
   scenario('full-page-modal', {
     // Hovering `body` to avoid random test failures
     // when "Open modal" sometimes is and sometimes is not hovered.
     actions: [click('#open-full-page-modal'), hover('body')],
     selectors: ['document'],
-    hideSelectors: ['body > *:not(.iui-modal)'],
+    hideSelectors: ['body > *:not(.iui-modal-backdrop)'],
+  }),
+
+  scenario('resizable-modal', {
+    // Hovering `body` to avoid random test failures
+    // when "Open modal" sometimes is and sometimes is not hovered.
+    actions: [click('#open-resizable-modal'), hover('body')],
```

Comment:
Tests fail: `waiting for selector `#open-resizable-modal` failed: timeout 30000ms exceeded` 

Change this to `toggle-resizable-modal` or change line 70 in modal.html.

Reply:
Thanks!